.github/workflows/issue_comment.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   notify:
     runs-on: ubuntu-latest
     permissions: {}
-    if: ${{ !github.event.issue.pull_request && !contains(fromJSON('["MEMBER", "OWNER"]'), github.event.comment.author_association) }}
+    if: ${{ !github.event.issue.pull_request }}
```

Comment:
I rather keep this filter to avoid sending notifications for comments from maintainers

Reply:
The if statement was moved down to each of the steps

Reply:
This `notify` job only has one step, which just runs the command. Unless i'm blatantly missing something 🤔

Reply:
Oh sorry, correct.

If you all would prefer, I can remove it. I brought this up with the Android team and we decided to remove it because we wanted to still be notified when a maintainer opened a new issue. This ensures everyone is aware of the issue coming in, not just the person that created it.

Reply:
But this action is for comments being added to issues, not for issues being opened.
I _also_ very much like not being notified for comments from us :P. 
But we can check with the team as well